package.json

```diff
@@ -5,17 +5,53 @@
   "repository": "git@github.com:sul-cidr/histonets.git",
   "author": "",
   "license": "Apache-2.0",
-  "dependencies": {},
+  "dependencies": {
+    "autoprefixer": "^7.1.1",
+    "babel-core": "^6.25.0",
```

Comment:
I should be able to move a bunch to devDependencies. Some of these were automatically placed here by the webpacker gem. I can try moving them and see if it breaks things.